File: Algo-DS-Python/Search/LinearSearch.py

# Linear Search

def linearSearch(arr, val):
    for i in range(len(arr)):
        if arr[i] == val:
            return True
    return False

print(linearSearch([1, 2, 3, 4, 5], 4)) # True

print(linearSearch([1, 2, 3, 4, 5], 99))  # False

File: DSA-JAVA/Searching/LinearSearch.java
public class LinearSearch {
    public static void main(String[] args) {
        int[] arr = {3, 2, 1, 6, 4, 5};
        int val = 6;

        for(int i=0; i<arr.length; i++){
            if(arr[i] == val){
                System.out.println(val+" found at: "+i);
                break;
            }
        }
    }
}
